### PR TITLE
Update function support page for 0.10.0-beta

### PR DESCRIPTION
**Pull request by ronkok**

### CODE REVIEW

File: function-support.html [hunk 1]
```diff
@@ -599,8 +598,16 @@ <h2 id="environments">Environments</h2>
...
+            <td>\(\def\arraystretch{1.5}\begin{array}{c|c:c} a & b & c \\ \hline d & e & f \\ \hdashline g & h & i \end{array}\)</td>
+            <td>
+                <pre><code>\def\arraystretch{1.5}
+\begin{array}{c|c:c}
+   a &amp; b &amp; c \\ \hline
+   d &amp; e &amp; f \\
+   \hdashline
+   g &amp; h &amp; i
+\end{array}</code></pre>
+            </td>
```
Comment (edemaine) [inline function-support.html]: Great example!

Technically, the various other `&`s above ought to be `&amp;` in HTML. But it's been working fine, so I could go either way.

File: function-support.html [hunk 2]
```diff
@@ -1214,14 +1225,43 @@ <h2 id="logic">Logic and Set Theory</h2>
...
+	<table class="symbol">
+		<tr>
+			<td>\(\def\foo{x^2}\foo + \foo\)</td>
+			<td><strong>\def</strong>\foo{x^2}&nbsp;\foo + \foo</td>
+		</tr>
+		<tr>
+			<td>\(\gdef\bar#1{#1^2}&nbsp;\bar{y} + \bar{y}\)</td>
+			<td><strong>\gdef</strong>\bar#1{#1^2}&nbsp;\bar{y} + \bar{y}</td>
+		</tr>
+		<tr> <td></td>  <td>\global\def\macroname[args]{definition}</td> </tr>
```
Comment (edemaine) [inline function-support.html]: I find this one a bit confusing, given that `[` actually means `[` in the examples below.  Perhaps replace `[args]` with `...` or `#1#2...`?

File: function-support.html [hunk 3]
```diff
@@ -1214,14 +1225,43 @@ <h2 id="logic">Logic and Set Theory</h2>
...
+		<tr>
+			<td>\mathchoice</td>   <td>\@ifnextchar</td>  <td>\relax</td>
+		</tr>
+		<tr>
+			<td>\TextOrMath</td>   <td>\@firstoftwo</td>  <td></td>
+		</tr>
+		<tr>
+			<td>\@ifstar</td>      <td>\@secondoftwo</td>  <td></td>
+		</tr>
+	</table>
```
Comment (edemaine) [inline function-support.html]: I could see going to 4 columns (at least) here. But not a big deal.

**Comment by ronkok** [comment]
Comments are picked up.

File: function-support.html [hunk 4]
```diff
@@ -1234,10 +1234,10 @@ <h2 id="macros">Macros</h2>
 			<td>\(\gdef\bar#1{#1^2}&nbsp;\bar{y} + \bar{y}\)</td>
 			<td><strong>\gdef</strong>\bar#1{#1^2}&nbsp;\bar{y} + \bar{y}</td>
 		</tr>
-		<tr> <td></td>  <td>\global\def\macroname[args]{definition}</td> </tr>
-		<tr> <td></td>  <td>\newcommand\macroname[args]{definition}</td>  </tr>
-		<tr> <td></td>  <td>\renewcommand\macroname[args]{definition}</td>  </tr>
-		<tr> <td></td>  <td>\providecommand\macroname[args]{definition}</td>  </tr>
```
Comment (edemaine) [inline function-support.html]: This is wrong: `\newcommand`, `\renewcommand`, `\providecommand` use `[numargs]` not `#1#2...`.

**Comment by kevinbarabash** [comment]
@edemaine sorry about.  You should be able to merge now.

**Pull request merged**